src/docs/reference/modules/Tenants/README.md

```diff
@@ -91,4 +91,19 @@ A feature profile can also be set when creating a tenant via the web API.
 
 ## Video
 
-<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/aQAjTG2ma64" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
\ No newline at end of file
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/aQAjTG2ma64" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Tenant Removal
```

Comment:
Can we move this before "Video" section, also we can add _By default, this feature is not allowed._ as note to make it clear for everyone

Reply:
Not sure as the Video is only related to (and at the end of) the previous section `Feature Profiles`.

> also we can add By default, this feature is not allowed

This is commented in the below sample.

    // Whether tenant removal is allowed, false by default.

Reply:
> Not sure as the Video is only related to (and at the end of) the previous section Feature Profiles.

The video section intend to be in the last, regardless of what's before

Reply:
Okay, I will move the video section to the end.